test/format/config.sh

```diff
@@ -246,6 +250,8 @@ CONFIG configuration_list[] = {
 
 {"runs.rows", "number of rows", C_TABLE, 10, M(1), M(100)}
 
+{"runs.predictable_replay", "configure predictable replay", C_BOOL, 0, 0, 0}
```

Comment:
Alphabetize in the list please

Reply:
This is fixed, I don't know why it doesn't say Outdated.